src/vs/workbench/browser/parts/media/compositepart.css

```diff
@@ -11,6 +11,24 @@
 	display: flex;
 }
 
+/* MEMBRANE: hide sidebar (Navigator) and panel (Logs) title */
```

Comment:
I forgot to mention this in our call but I really like the removal of the tabs on the bottom panel 👌